### ISSUE TITLE
Rich Text Format to About Page

### ISSUE THREAD

**Issue by thomcrowe** [manuelernestog/astropod#10]
Is there an easy way to add rich text format to the about page content in the general section of the CMS?

**Comment by oceanpenguin** [comment]
I agree this would be a useful feature for me. For consistency it would make sense if the formatting worked the same as with episode notes, with markdown.
